packages/jack-dummy/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://jackaudio.org/
+TERMUX_PKG_DESCRIPTION="jack1 (dummy module only)"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_LICENSE="GPL-2.0, LGPL-2.0"
+TERMUX_PKG_DEPENDS="libdb, libandroid-sysv-semaphore, libandroid-shmem"
+TERMUX_PKG_VERSION=0.126.0
+TERMUX_PKG_GIT_BRANCH=$TERMUX_PKG_VERSION
+TERMUX_PKG_SRCURL=git+https://github.com/jackaudio/jack1
+TERMUX_PKG_AUTO_UPDATE=true
+
+termux_step_pre_configure() {
+	cd ${TERMUX_PKG_SRCDIR}
+	git submodule update --init --recursive
```

Comment:
You dont need to manually `git submodule update`. Its done in `termux_git_clone_src` when `TERMUX_PKG_SRCURL` is a git repo. https://github.com/termux/termux-packages/blob/932cb3ea25991ed9020d1e4ca98c37b9e9f2a705/scripts/build/get_source/termux_git_clone_src.sh#L17

Also please remove the repeating `&&`.